Odata-docs/client/using-dataservice-ihttpclientfactory.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Using DataserviceContext IHttpClientFactory in Microsoft.OData.Client"
+description: "This tutorial provides guidance on using IHttpClientFactory with OData Client in a .NET application."
+
+author: Wanjohi Sammy
+ms.author: swanjohi
+ms.date: 11/29/2024
+ms.topic: article
+ 
+---
+
+# Using DataserviceContext IHttpClientFactory in Microsoft.OData.Client
+
+Ever wanted to use `HttpClient` with [`Microsoft.OData.Client`](/odata/client/getting-started)? Now you can! In this post, we'll show you how to do it easily and highlight the benefits.
```

Comment:
Note that OData Client uses `HttpClient` by default. What `IHttpClientFactory` lets you do is to provide and configure a custom instance of `HttpClient` for OData client to use. However this paragraph and other parts of this doc may be read as suggesting that `IHttpClientFactory` is required for OData Client to use `HttpClient`.